run_ci.sh

```diff
@@ -14,6 +14,10 @@ function getlogs(){
     sudo podman logs coreos-downloader > $LOGDIR/coreos-downloader.log
     sudo podman logs ipa-downloader > $LOGDIR/ipa-downloader.log
 
+    sudo cp /var/log/virtualbmc/virtualbmc.log $LOGDIR/bootstrap-host-virtualbmc.log
```

Comment:
I think this log location just changed -- https://github.com/metal3-io/metal3-dev-env/pull/48/files

Reply:
Hmm that may explain why vbmc is broken in our CI then ;)

Reply:
I guess we need to start the containers as in 02_configure_host.sh from metal3-dev-env/pull/48 and I don't see any corresponding dev-scripts PR

I'll test locally and push one.